pokemongo_bot/cell_workers/catch_lured_pokemon.py

```diff
@@ -12,39 +13,52 @@ class CatchLuredPokemon(BaseTask):
     def work(self):
         lured_pokemon = self.get_lured_pokemon()
         if lured_pokemon:
-            self.catch_pokemon(lured_pokemon)
+            for pokemon in lured_pokemon:
```

Comment:
Wow, it looks like the code that was here was really bad. It only checked the nearest fort for lured pokemon? That is totally broken. It looks like the new code goes through all nearby forts to find all lured pokemon. Could we take that win separately from catching all the pokemon here?

I'm proposing we just leave this `work()` function the way it is in this PR. We will get lots of improvements from the fix to `get_lured_pokemon`. It would be good to see how much of the behavior we need that is.